script/Arcueid/Day 13/13_00_ARC13_1.txt

```diff
@@ -821,12 +821,12 @@ You could never have succeeded the Tohno family in the first place.
 [sha:1f9740f73baec8b6c335d194614e709604c354d3]{
 -- Page 55, Offset 31825. Mods: @n.
 -- 本当にそんな事さえ忘れてくれていたんですね。
-To think that you've managed to forget even that
+To think that you've managed to forget even that...
 }
 [sha:296153b5ac049cb48510eacb5052856468765762]{
 -- Page 55, Offset 31826. Glued. Mods: @k, @e.
 -- ……
-...
```

Comment:
dont edit it like that. i think it might cause problems

Reply:
I want it to pause after "that..."

Reply:
the ellipsis will appear like that in game
that...

Reply:
I know, but there was a pause before the three dots in the game, "To think that you've managed to forget even that" _short pause_ "..." and the next line follows

Reply:
thats just how the engine works i guess

Reply:
You can do it, you just need to 
1. Change `sha:296153b5ac049cb48510eacb5052856468765762` here to `offset:31826` 
2. Change the translation of the line to be `%{nothing}`, if it's just the empty string deepLuna doesn't count it as translated